core/sets.gms

```diff
@@ -753,6 +750,9 @@ LAM_regi,OAS_regi,SSA_regi,EUR_regi,NEU_regi,MEA_regi,REF_regi,CAZ_regi,CHA_regi
 LAM,OAS,SSA,EUR,NEU,MEA,REF,CAZ,CHA,IND,JPN,USA
       /
  
+
+   all_regi "all regions" /LAM,OAS,SSA,EUR,NEU,MEA,REF,CAZ,CHA,IND,JPN,USA/
+
```

Comment:
> - submit the moving up of the `all_regi` stuff that was introduced [here](https://github.com/remindmodel/remind/pull/1120/files#diff-c328a7ad81b0c7c5467604c39f07ba8f2cf20e165d7a72df04abf175ddd58e8a)

I fail to see how this was "introduced", not what effect it is supposed to have …

Reply:
Well, this is the part of `core/sets.gms` that is written by `prepare_and_run`. Felix moved the `all_regi` stuff a bit down, and so if you execute `prepare_and_run`, then `core/sets.gms` is changed. And I just committed this change so a usual H12 run doesn't change `core/sets.gms` anymore. Was indeed unrelated to the NDC error, just some cleanup.